src/screens/debug/debug.tsx

```diff
@@ -1,21 +1,57 @@
-import React, { FC } from 'react';
+import React, { FC, useState } from 'react';
+import { useDispatch } from 'react-redux';
 
 import { ButtonMedium } from '../../components/button/button-medium/button-medium';
 import { Divider } from '../../components/divider/divider';
 import { IconNameEnum } from '../../components/icon/icon-name.enum';
 import { ScreenContainer } from '../../components/screen-container/screen-container';
+import { StyledTextInput } from '../../components/styled-text-input/styled-text-input';
+import { AccountTypeEnum } from '../../enums/account-type.enum';
+import { useNavigation } from '../../navigator/hooks/use-navigation.hook';
 import { useRecentActionsSelector } from '../../store/debug/debug-selectors';
+import { addHdAccountAction, setSelectedAccountAction } from '../../store/wallet/wallet-actions';
+import { formatSize } from '../../styles/format-size';
+import { showSuccessToast } from '../../toast/toast.utils';
 import { ActionItem } from './action-item/action-item';
 
 export const Debug: FC = () => {
   const recentActions = useRecentActionsSelector();
+  const dispatch = useDispatch();
+  const { goBack } = useNavigation();
+
+  const [name, setName] = useState('');
+  const [publicKeyHash, setPublicKeyHash] = useState('');
+
+  const publicData = {
+    name,
+    type: AccountTypeEnum.WATCH_ONLY_DEBUG,
+    publicKey: 'publicKey',
+    publicKeyHash
+  };
```

Comment:
move `publicData` into `handleImportButtonPress`  
validate that `publicKeyHash` is valid address